src/isAnnotatedForRemoval.js

```diff
@@ -0,0 +1,7 @@
+// @flow weak
+
+export default function (node) {
+  const comments = node.trailingComments || [];
```

Comment:
I feel better with `trailingComments` (a la [flow](https://flow.org/blog/2015/02/20/Flow-Comments/)).

A minor flaw with `leadingComments` could occur when the declaration is [inlined](http://astexplorer.net/#/gist/593c06903a408bfff33d09db9e38144b/e2a7a6c559538427bba6e5ed025b9a7125923c16).
And I'm not sure how to search for comments this way, as opposed to leading the LHS.